whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -17,11 +17,11 @@
     <!-- @Transactional CAN be used on private methods with mode="aspectj". However, annotations on interfaces will NOT work! -->
     <tx:annotation-driven mode="aspectj"/>
 
-    <context:spring-configured/>
+    <context:spring-configured />
```

Comment:
How do we replace `context:spring-configured` ?

Reply:
@EnableSpringConfigured

Reply:
Done